template-parts/header-personal.php

```diff
@@ -9,7 +9,7 @@
 					if ( function_exists( 'the_custom_logo' ) && has_custom_logo() ) {
 						the_custom_logo();
 					} else {
-						echo '<a href="<?php echo get_home_url(); ?>">';
+						echo '<a href="'get_home_url(); '">';
```

Comment:
```suggestion
						echo '<a href="' . get_home_url() . '">';
```
Ich denke, das muss so...